packages/theme/src/themes/dark.js

```diff
@@ -10,7 +10,7 @@ import colors from '../colors';
 
 const darkTheme = {
   primary: colors.gray1,
-  primaryInverse: colors.gray6,
+  primaryInverse: colors.black,
```

Comment:
This had to be changed to match the old ui-framework style.

Reply:
this change should live in its own `fix` commit

Reply:
okey moved it to: https://github.com/quid/refraction/pull/50